content/docs/0.6.0/usecases/self-healing-feature-flags/index.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Self-healing with Feature Flags
+description: Demonstrates how to use the self-healing mechanisms of Keptn to automatically set feature flags in an Unleash feature flag server.
+weight: 40
+keywords: [self-healing]
+aliases:
+---
+Demonstrates how to use the self-healing mechanisms of Keptn to automatically set feature flags in an Unleash feature flag server.
+
+## About this tutorial
+
+In this tutorial you will learn how to use the capabilities of Keptn to provide self-healing for an application with feature flags based on the [Unleash feature toggle framework](https://unleash.github.io/).
+
+## Prerequisites
+
+- Finish the [Onboarding a Service](../onboard-carts-service/) tutorial.
+
+- Clone the example repository, which contains specification files:
+
+    ```console
+    git clone --branch 0.6.0.beta2 https://github.com/keptn/examples.git --single-branch
+    ```
+
+- [Setup monitoring with Dynatrace](../../reference/monitoring/dynatrace/)
+
+- Access to an Unleash server with an access token (see below).
+
+## Onboard and deploy the Unleash server with Keptn
+
+**Note**: You can skip this step if you already have access to an Unleash server. 
+
+To quickly get an Unleash server up and running with Keptn, follow the instructions provided here: https://github.com/keptn/examples/tree/master/unleash-server .
```

Comment:
I would prefer showing the install steps for the Unleash server here instead of referring to an external repo.

Reply:
I was unsure whether we would want to have the installation-steps here as well, as it would be redundant. I'll include them.

Reply:
done